### PR TITLE
added web tests

### PR DESCRIPTION
**Pull request by freglyc**
Added base cases that test the OpenStax webpage.

### CODE REVIEW

File: tests/web/test_web_about_openstax_tutor.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+
```
Comment (gregfitch) [inline tests/web/test_web_about_openstax_tutor.py]: Missing the file docstring and imports:
`NameError: name 'test_case' is not defined`
`NameError: name 'web' is not defined`
etc.

File: tests/web/test_web_about_openstax_tutor.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+@test_case('')
+@web
+def test_view_about_tutor(web_base_url, selenium):
+    """Tests ability to view about OpenStax tutor page."""
+    
+    # GIVEN: On the OpenStax homepage
+
+    # WHEN: Hover over "Technology"
+
+    # AND: Click on "About OpenStax Tutor"
```
Comment (gregfitch) [inline tests/web/test_web_about_openstax_tutor.py]: `GIVEN` doesn't need a blank line before it; likewise, the `AND` is a part of the `WHEN` clause and doesn't need a blank line between them.

File: tests/web/test_web_about_openstax_tutor.py (hunk 1 above)
Comment (freglyc) [reply tests/web/test_web_about_openstax_tutor.py]: Updated

**Comment by gregfitch** [comment]
careful... it's `tests.markers`

**Comment by gregfitch** [comment]
Make sure `tox -- --driver chrome -n 5 tests/web/*.py` can complete without Flake errors

**Comment by gregfitch** [comment]
Woohoo! It passed!

File: tests/web/test_web_admin.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    # AND: Click OpenStax Homepage.
+    # AND: Click on OpenStax Support.
+    # AND: Fill in required fields and click publish.
+
+    # THEN: Support page is updated with CMS content.
+
+
+@test_case('')
+@expected_failure
+@nondestructive
```
Comment (gregfitch) [inline tests/web/test_web_admin.py]: Watch out for these; this test (editing and/or adding a book) will definitely be destructive.

Comment (freglyc) [reply tests/web/test_web_admin.py]: Whoops. They should be fixed now!

**Pull request merged**